vendor.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/creack/pty v1.1.21
 	github.com/distribution/reference v0.5.0
 	github.com/docker/distribution v2.8.3+incompatible
-	github.com/docker/docker v25.0.3+incompatible
+	github.com/docker/docker v25.0.4-0.20240301160236-51e876cd964c+incompatible
```

Comment:
Perhaps good to add a comment here to indicate where this version came from, e.g. something like;

```
// 25.0 branch (v25.0.4-dev)
```